src/components/Landing/ConnectAndCreateButton.tsx

```diff
@@ -0,0 +1,75 @@
+import { displayFrom, displayTo } from 'helpers/visibilityClassnames'
+import { handleError } from '@big-whale-labs/frontend-utils'
+import { useLocation } from 'wouter'
+import BurnerWalletStore from 'stores/BurnerWalletStore'
+import Button from 'components/ui/Button'
+import getErrorMessage from 'helpers/getErrorMessage'
+import walletStore from 'stores/WalletStore'
+
+const defaultMessage =
+  'We could not match the username with the wallet you have connected. Please review them and try again.'
+
+export default function ({
+  account,
+  loading,
+  burnerLoading,
+  onError,
+  onLoading,
+}: {
+  account?: string
+  loading: boolean
+  burnerLoading: boolean
+  onError: (error: string) => void
+  onLoading: (loading: boolean) => void
+}) {
+  const [, setLocation] = useLocation()
+
+  const createBurnerWallet = async () => {
+    onError('')
+    onLoading(true)
+
+    try {
+      if (!walletStore.account) await walletStore.connect(true)
```

Comment:
Why don't we use `const {account} = useSnapshot(WalletStore)` here?
Or at least you can add a
`const {account} = walletStore`
And use `account` instead of `WalletStore.account`

Reply:
Because of the naming you can do:
```TS
const { account: UserAccount } = walletStore
```

Reply:
Changed